mira/constants.py

```diff
@@ -0,0 +1,34 @@
+"""Constants used across MIRA."""
+
+import pystow
+
+__all__ = ["MODULE", "NODE_HEADER", "EDGE_HEADER"]
+
+#: The PyStow module which allows for system-independent
+#: creation of stable file paths.
+MODULE = pystow.module("mira")
```

Comment:
This creates  a folder when the module is loaded and makes pystow a base requirement, not sure it's necessary though unless we're dealing with a specific workflow that creates temporary files.